dispatch/static/manager/src/js/vendor/dispatch-editor/components/Editor.js

```diff
@@ -88,13 +89,13 @@ class ContentEditor extends React.Component {
   componentWillReceiveProps(nextProps) {
     // Push content to editor state
     this.setState({
-      editorState: EditorState.push(this.state.editorState, nextProps.content)
+      editorState: EditorState.push(this.state.editorState, nextProps.content || ContentState.createFromText(''))
```

Comment:
When I tried that, having undefined/null content state caused various other errors.

In particular toJSON would need a special case. 
Is that preferable?